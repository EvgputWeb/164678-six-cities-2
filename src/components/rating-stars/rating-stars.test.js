import React from 'react';
import renderer from 'react-test-renderer';
import RatingStars from './rating-stars';


it(`RatingStars renders correctly`, () => {
  const review = renderer
    .create(
        <RatingStars
          classPrefix={`property`}
          rating={5}
          isValueVisible={true}
        />)
    .toJSON();

  expect(review).toMatchSnapshot();
});
