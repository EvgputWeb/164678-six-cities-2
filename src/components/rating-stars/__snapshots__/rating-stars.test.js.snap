// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingStars renders correctly 1`] = `
<div
  className="property__rating rating"
>
  <div
    className="property__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "100%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
  <span
    className="property__rating-value rating__value"
  >
    5.0
  </span>
</div>
`;
